Jenkinsfile

```diff
@@ -95,6 +95,8 @@ pipeline {
           RSYNC_PASSWORD=\$(cat \$keyfile)
           export RSYNC_PASSWORD=\${RSYNC_PASSWORD:0:13}
           set -x
+          # Change perms to allow reading on webserver side
+          chmod -R o+r ./builds/
```

Comment:
Probably also worth doing a `chmod a+rX`. See e.g. https://github.com/CentOS/sig-atomic-buildscripts/pull/351. So something like:

```
find builds/ ! -type l -exec chmod a+rX {} +
```

Reply:
the PR you linked to did `-type d`. Did you mean to say `-type l` ?

Reply:
See https://github.com/CentOS/sig-atomic-buildscripts/pull/355.

Reply:
fixed